metaci/plan/migrations/0040_plan_commit_status_regex.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.1.13 on 2021-08-09 16:52
+
+from django.db import migrations, models
+
+
+def migrate_regexes(apps, schema_editor):
+    Plan = apps.get_model("plan", "Plan")
+    for plan in Plan.objects.all():
+        if plan.trigger == "status" and plan.regex:
```

Comment:
We can avoid loading all of the plans from the db:

```suggestion
    for plan in Plan.objects.filter(trigger="status").iterator():
        if plan.regex:
```